src/system/device.js

```diff
@@ -22,7 +26,12 @@
         var swipeEnabled = true;
         var disableSwipeFn = function (e) {
             e.preventDefault();
-            window.scroll(0, 0);
+            if (!me.device.isWxSmallGame)
+            {
+                // There is not DOM in WxSmallGame runtime.
+                window.scroll(0, 0);
+            }
+            
```

Comment:
since disableSwipeFn is a user called function, I don't see the need to add this check here (melonJS never calls it himself). And the test should rather be in your game to call it or not if you need this on a standard device/browser